### PR TITLE
Add ferdy flip testnet faucet config

### PR DESCRIPTION
**Pull request by tactical-retreat**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -180,6 +180,23 @@
...
                 "WINDOW_SIZE": 1440
             }
+        },
+        {
+            "ID": "FERDYNET",
+            "NAME": "FerdyFlip Testnet",
+            "TOKEN": "FAVAX",
+            "RPC": "https://testnet-rpc.ferdyflip.xyz/rpc",
+            "CHAINID": 668577,
+            "EXPLORER": "https://devnet.routescan.io",
```
Comment (rajranjan0608) [inline config.json:190]: Can we have actual explorer link here, so that users can view transaction details for their faucet drip?

**Review by rajranjan0608 — APPROVED** [review]

File: config.json (hunk 1 above)
Comment (tactical-retreat) [reply config.json:190]: We don't have an explorer yet. I'm trying to make sure that things work before I ask for integrations, but it's hard to make sure things work without being able to get funds to them so people can test.

I can send a follow-up PR once it's available? Or is this a hard requirement.

Comment (rajranjan0608) [reply config.json:190]: Cool. Np, it's just that users won't be able to see drip receipt, otherwise shouldn't be a blocker 👍

Comment (tactical-retreat) [reply config.json:190]: Thanks! Will send a follow up when available.

**Pull request merged**